addons/account/models/account.py

```diff
@@ -702,6 +702,18 @@ def copy(self, default=None):
         default = dict(default or {}, name=_("%s (Copy)") % self.name)
         return super(AccountTax, self).copy(default=default)
 
+    @api.multi
+    def name_get(self):
+        if not self._context.get('include_taxes_type'):
+            return super(AccountTax, self).name_get()
+        result = []
+        type_tax_use_selection = dict(self._fields['type_tax_use'].selection)
```

Comment:
In this case, do you know a better way to retrieve the translated terms of such selection field plz?